extensions/binding/org.eclipse.smarthome.binding.coapconnectedlighting/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Bundle-ClassPath: .,
  lib/element-connector-1.1.0-SNAPSHOT.jar
 Import-Package: 
  org.apache.commons.lang;version="2.6.0",
+ org.eclipse.jdt.annotation,
```

Comment:
Please make this dependency optional, see: https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/archetype-resources/META-INF/MANIFEST.MF#L16